interface/new/new_comprehensive.php

```diff
@@ -849,13 +835,27 @@ function end_group()
 // var override = false; // flag that overrides the duplication warning
 
 $(document).ready(function() {
-enable_modals();
+/*enable_modals();
  $(".medium_modal").fancybox( {
                 'overlayOpacity' : 0.0,
                 'showCloseButton' : true,
                 'frameHeight' : 460,
                 'frameWidth' : 650
+        });*/
```

Comment:
delete above (i am sure you are planning to, but my obsessive code review disease makes me type this)